lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -794,14 +795,52 @@ public void playAnimation() {
       }
     }
     if (!animationsEnabled()) {
-      setFrame((int) (getSpeed() < 0 ? getMinFrame() : getMaxFrame()));
+      Marker markerForAnimationsDisabled = getMarkerForAnimationsDisabled();
+      if (markerForAnimationsDisabled != null) {
+        setFrame((int) markerForAnimationsDisabled.startFrame);
```

Comment:
On iOS we also support reduced motion markers with both a start frame and an end frame. If the marker provides an end frame, we play the animation from the start / end of the reduced motion marker as opposed to just only displaying the first frame.

Reply:
@calda  Thanks for the quick reply, Is there a way to play from the start to end frame? Sorry for not knowing from looking at this code.

Reply:
@gpeal should be able to help :)

Reply:
@gpeal  How would I play the animation from start to end here per @calda 's comment?